src/bootstrap/configure.py

```diff
@@ -521,13 +521,19 @@ def write_config_toml(writer, section_order, targets, sections):
         else:
             writer = write_uncommented(sections[section], writer)
 
+def quit_if_file_exists(file):
+    if os.path.isfile(file):
+        p("Existing '" + file + "' detected. EXITING")
+        quit()
 
 if __name__ == "__main__":
     p("processing command line")
     # Parse all known arguments into a configuration structure that reflects the
     # TOML we're going to write out
     p("")
     section_order, sections, targets = parse_args(sys.argv[1:])
+    # If 'config.toml' already exists, exit the script at this point
+    quit_if_file_exists('config.toml')
```

Comment:
nit: why only exit here? I think it would be nice to put this at the very start of the block, right now the output looks like this:
```
; ./configure 
configure: processing command line
configure: 
configure: build.configure-args := []
configure: Existing 'config.toml' detected. EXITING
```